src/TwitchLib.Communication/Enums/MessageRateLimit.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace TwitchLib.Communication.Enums
+{
+    /// <summary>
+    ///     <see href="https://dev.twitch.tv/docs/irc/#rate-limits"/>
+    /// </summary>
+    public enum MessageRateLimit : uint
+    {
+        Limit_20_in_30_Seconds = 20,
+        Limit_100_in_30_Seconds = 100,
+        Limit_7500_in_30_Seconds = 7_500,
```

Comment:
typo? 7_500

Reply:
It's a notation, that makes large numbers more readable.
The underscore can be read like a thousand separator.

Reply:
It's "new" thing from C# 7.0, it's great, especially when dealing with large numbers. The compiler basically ignores those underscores, but for us humans it's much more readable.